### PR TITLE
Added 929003055601 (4096730P6)

### PR DESCRIPTION
**Pull request by RonnyAL**
Different zigbeeModel than the existing entry. Seen in Norway.

### CODE REVIEW

File: devices/philips.js [hunk 1]
```diff
@@ -1220,6 +1220,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['929003055601'], // The model ID from: Device with modelID 'lumi.sens' is not supported.
```
Comment (sjorge) [inline devices/philips.js:1224]: Drop the comment on this line please.

Comment (RonnyAL) [reply devices/philips.js]: Done. I kept the original commit message though I'm not sure that's optimal. Again, I'm pretty new to this part of git.

Comment (sjorge) [reply devices/philips.js]: Koen squashes the commits when merging so it's not a problem.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**